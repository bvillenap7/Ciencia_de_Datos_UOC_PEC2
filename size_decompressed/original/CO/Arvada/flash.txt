it was a flourescent green with red on the front and a long tail.;1999-02-01 22:05:00;5 seconds;it was a flourescent green with red on the front and a long tail. sky was partly cloudy it traveled from south to north at about 10:00in sky it traveled fast then disappeared;39.8289982352941;-105.110888235294
Three 3 sec enormous oval shaped football field size flashes in sky w/intervals of 5sec That radiated out to entire sky which  lit sky.;2019-01-11 21:30:00;3-5 sec ea,15 sec total;three 3sec enormous oval shaped football field size flashes in sky w/intervals of 5sec That radiated out to entire sky which  lit up the sky bright white center then flashed to blue-yellow;39.8289982352941;-105.110888235294
