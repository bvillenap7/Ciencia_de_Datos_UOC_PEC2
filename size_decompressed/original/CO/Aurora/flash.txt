As i was walking facing west, i began to notice a star like flash. It happened very suddenly,then i was walking to my door to the house;2002-10-21 09:15:00;5sec.;As i was walking facing west, i began to notice a star like flash. It happened very suddenly,then i was walking to my door to the house...i noticed it again.I have seen many types of U.F.O.s in my 24 yrs of life. I also investigate the sightings here in Colorado. Mostly during the morning through afternoon hours. I also have seen this type before.Does anyone have an answer on what those flashes are??? This all took place at 9pm Monday night. If anyone had seen the flashes...they should contact this web.site.The color of the flashes were like a blue colored star.;39.6695444272445;-104.803991486068
I'm watching this green and blue light flash its been raising altitude as a time passes the only light flashing in the sky how to your;2015-10-13 03:17:00;40 minute +;I'm watching this green and blue light flash its been raising altitude as a time passes the only light flashing in the sky how to your interview the light aircraft just hovers over Aurora. ((NUFORC Note:  Witness elects to remain totally anonymous provides no contact information.  PD));39.6695444272445;-104.803991486068
