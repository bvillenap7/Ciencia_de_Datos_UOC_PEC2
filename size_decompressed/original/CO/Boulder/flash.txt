Light flash;2008-06-27 20:00:00;10 seconds;Just was hanging out in the park and noticed a light flash, kind of like a mirror reflecting light in the dusk sky.  It completely disappeared right away. ((NUFORC Note:  Witness elects to remain totally anonymous  provides no contact information.  PD));40.0217727272726;-105.258365102639
