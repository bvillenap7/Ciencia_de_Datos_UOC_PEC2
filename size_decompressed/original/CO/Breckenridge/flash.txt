((NUFORC Note:  Witness provides no information elects to remain totally anonymous.  PD));2017-05-13 19:00:00;~10-20 minutes;((NUFORC Note:  Witness provides no information elects to remain totally anonymous.  PD));39.4844;-106.0192
