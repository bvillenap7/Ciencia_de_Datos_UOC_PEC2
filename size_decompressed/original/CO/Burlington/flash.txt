An ear-splitting almost explosion-type burst that left the whole horizon a luminous semi-circle of purple;2007-12-05 05:30:00;5 seconds;We didn't see a UFO or anything, this was an ear-splitting almost explosion-type burst that left the whole horizon a luminous semi-circle of purple.   We had been driving through a snowstorm it was still snowing so we didn't feel that it could have been lightning we saw nothing like lightning, only heard a gigantic wave of sound similar to the thunder accompanied by lightning, then saw the entire horizon light up in a bright purple hue.  My 3 children and I were just getting off of I-70 west heading to a Denny's at which the waitress admitted she had been witness to the same.;39.3133;-102.2614
Nearing Burlington, CO predawn on Dec 8, 2007 the entire horizon lit up in a huge semi-circular flash of purple;2007-12-08 05:30:00;5 seconds;Nearing Burlington, CO predawn on Dec 8, 2007 the entire horizon lit up in a huge semi-circular flash of purple We did not actually see a UFO, however, my 3 children and I were driving west on I-70 through a snowstorm nearing the Burlington exit when suddenly there was a loud (ear-splitting) burst similar to the thunder that accompanies a lightning storm the entire horizon lit up in a huge semi-circular flash of purple light.   We wondered if it was an explosion, some power lines having fallen or something, but saw nothing afterwards such as fire or emergency vehicles.   When we arrived at the Denny's shortly thereafter, the waitress admitted she had witnessed the exact same phenomenon.;39.3133;-102.2614
