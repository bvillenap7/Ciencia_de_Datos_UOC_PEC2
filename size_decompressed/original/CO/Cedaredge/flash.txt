MULTIPLE INTERMITENT FLASHES OF LIGHT;2016-03-26 23:15:00;5 minutes;MULTIPLE INTERMITENT FLASHES OF LIGHT MEMBERS OF THIS FAMILY HAVE SEEN THESE BEFORE SUMMER OF 2015, WINTER OF 2016 MULTIPLE SIGHTINGS FLASHES OF LIGHT APPEARED IN THE CLEAR NIGHT SKY LOOKING NORTH OVER THE GRAND MESA   MY BROTHER SAW APPARENTLY 10 DIFFERENT FLASHES FROM TWO NORTHERLY LOCATIONS  HE CAME INTO THE HOUSE AND ASKED ME TO COME LOOK AT WHAT HE WAS SEEING.  WHEN I CAME OUT I SAW THEM AS WELL - I SAW AT LEAST 6 FLASHES FROM AT LEAST ONE MAYBE TWO SOURCES FROM ONE OF THE REGIONS HE SHOWED ME HE SAW THEM APPEAR.  -  THE FLASHES WE BOTH WITNESSED WERE INTERMITTENT... -  FLASHES WERE AS BRIGHT OR SLIGHTLY BRIGHTER THAN THE BRIGHTEST STARS IN THE SKY -  THEY VARIED IN DURATION - SOME WERE VERY QUICK FLASH LESS THAN A SECOND - -  OTHERS SLOWLY LIT UP AND DIMMED OVER A PERIOD OF A 2-3 SECONDS. -  THE QUICK FLASHES WERE WHITE - THE SLOWER ONES WERE BURNT ORANGE COLOR -  THEY DID NOT FLASH FROM A LINEAR TRAJECTORY - THEY APPEARED RANDOMLY  THE ENTIRE SIGHTING ON THIS NIGHT LASTED APPROXIMATELY 5 MINUTES TOTAL - WHEN THEY STOPPED WE KEPT LOOKING FOR ABOUT 10 MINUTES THEN WE WENT INSIDE - UNAWARE IF THEY CONTINUED.;38.9141;-107.9241
