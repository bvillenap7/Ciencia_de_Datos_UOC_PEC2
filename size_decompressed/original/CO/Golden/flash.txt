Flashing metallic objects not exhibiting typical flight charactaristics of any known aircraft I or anyone else there has ever seen.;2008-11-03 13:10:00;15 minutes;Flashing metallic objects not exhibiting typical flight charactaristics of any known aircraft I or anyone else there has ever seen. At approximately 1:15pm on 11/3/08 I was outside and looked up and saw three metallic-like objects traveling at the same speed heading North.  After a couple of minutes, 2 of the objects abruptly stopped int heir tracks and the third continued north and that one subsequently came to an abrupt hault about thirty seconds later.   About a minute after all three were stationary, the first two that had stopped suddenly began to in circular motions and zig-zag motions, moving quickly in one direction then stopping quickly, then suddenly accellerating in another direction and stopping abriptly again.  This dancing lasted for roughly 5 minutes.   Then a cluster of four more identical objects appeared in the sky just south of the other three and were traveling in what some might call a formation.  almost as if they were connected.  But then they began to act independently swooping up and down, left and right as if they were in a dogfighting scene out from the movie Top Gun.   They then disappeared into the cirus clouds and did not reappear after watching the sky for another 10 minutes. ((NUFORC Note:  Witness elects to remain totally anonymous  provides no contact information.  PD));39.7598265306123;-105.220002040816
Bright white strobe light.;2015-06-07 23:30:00;15 minutes;Bright white strobe light. Observed bright white flashing light similar to an aircraft strobe light.  Occurred as 3 single flashes about 5 to 10 minutes apart approximately 5 to 10 degrees east of the cup of the big dipper in the northern shy.  The flashes did not appear to change location in the 15 minutes observed.;39.7598265306123;-105.220002040816
